packages/components/src/table/index.js

```diff
@@ -300,39 +300,41 @@ class TableCard extends Component {
 							type={ searchBy }
 						/>
 					),
-					( downloadable || onClickDownload ) && (
-						<IconButton
-							key="download"
-							className="woocommerce-table__download-button"
-							disabled={ isLoading }
-							onClick={ this.onClickDownload }
-							isLink
-						>
-							<DowloadIcon />
-							<span className="woocommerce-table__download-button__label">
-								{ labels.downloadButton || __( 'Download', 'wc-admin' ) }
-							</span>
-						</IconButton>
-					),
 				] }
 				menu={
```

Comment:
Thinking out loud, I wonder if we should rename this prop if we are going to use it to display other components than the menu itself.

Related to that, I get a JS error because of a PropType mismatch in `<Card>`:
`Warning: Failed prop type: Invalid menu passed to Card. Must be EllipsisMenu`

Reply:
Updated the prop type and description here to fix the error.

>Thinking out loud, I wonder if we should rename this prop if we are going to use it to display other components than the menu itself.

I had a similar thought and then came to the conclusion that `menu` was still a descriptive area to where it was being displayed and actions can be a part of a menu area.  That said, I'm open to other names here if anyone has suggestions.  Maybe we change this to `menuArea` and `actionArea` or `leftAction` and `rightAction` (though that would be inaccurate on mobile).

Reply:
> I had a similar thought and then came to the conclusion that `menu` was still a descriptive area to where it was being displayed and actions can be a part of a menu area. That said, I'm open to other names here if anyone has suggestions. Maybe we change this to `menuArea` and `actionArea` or `leftAction` and `rightAction` (though that would be inaccurate on mobile).

Fair enough.